google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -3322,4 +3322,37 @@ public void testDeleteLifecycleRules() throws ExecutionException, InterruptedExc
       RemoteStorageHelper.forceDelete(storage, bucketName, 5, TimeUnit.SECONDS);
     }
   }
+
+  @Test
+  public void testUpdateBucketLogging() throws ExecutionException, InterruptedException {
+    String logsBucket = RemoteStorageHelper.generateBucketName();
+    String loggingBucket = RemoteStorageHelper.generateBucketName();
+    try {
+      assertNotNull(storage.create(BucketInfo.newBuilder(logsBucket).setLocation("us").build()));
+      Policy policy = storage.getIamPolicy(logsBucket);
+      assertNotNull(
+          storage.setIamPolicy(
+              logsBucket,
+              policy
+                  .toBuilder()
+                  .addIdentity(StorageRoles.legacyBucketWriter(), Identity.allAuthenticatedUsers())
+                  .build()));
+      BucketInfo.Logging logging =
+          BucketInfo.Logging.newBuilder()
+              .setLogBucket(logsBucket)
+              .setLogObjectPrefix("test-logs")
+              .build();
+      Bucket bucket =
+          storage.create(
+              BucketInfo.newBuilder(loggingBucket).setLocation("us").setLogging(logging).build());
+
+      assertEquals(logsBucket, bucket.getLogging().getLogBucket());
+      assertEquals("test-logs", bucket.getLogging().getLogObjectPrefix());
+      Bucket updatedBucket = bucket.toBuilder().setLogging(null).build().update();
+      assertThat(updatedBucket.getLogging()).isNull();
+    } finally {
+      RemoteStorageHelper.forceDelete(storage, logsBucket, 5, TimeUnit.SECONDS);
+      RemoteStorageHelper.forceDelete(storage, loggingBucket, 5, TimeUnit.SECONDS);
+    }
+  }
```

Comment:
You duplicated the code of an existed tests, that is not good. 
Could you extend the existing test to check that set loggin to null have an effect, please